cogs/moderation/moderation.py

```diff
@@ -118,9 +117,8 @@ async def tempban(
         length: CustomTimeConverter,
         *,
         reason="No Reason Provided",
-    ):
-
-        # TODO: add comments and docstrings
+    ) -> None:
```

Comment:
Why's there a `*` here btw? We declare arguments to be keyword-only in function definitions, not function calls.

Reply:
This is a function definition?

Reply:
My bad github hiding lines in between was too much for my smol brain to handle